### PR TITLE
[FIX] fetchmail: using undeclared variable

### PR DESCRIPTION
**Pull request by yajo**
When there are no emails to fetch, you get this error:

```
INFO odoo odoo.addons.fetchmail.models.fetchmail: start checking for new emails on pop server XXXX 
INFO odoo odoo.addons.fetchmail.models.fetchmail: General failure when trying to fetch mail from pop server XXXX.                                                                                                                                                                                                          
Traceback (most recent call last):       
  File "/opt/odoo/auto/addons/fetchmail/models/fetchmail.py", line 210, in fetch_mail                                                         
    _logger.info("Fetched %d email(s) on %s server %s; %d succeeded, %d failed.", num, server.server_type, server.name, (num - failed_in_loop), failed_in_loop)         
UnboundLocalError: local variable 'num' referenced before assignment                                                                                 
```

The fix is pretty obvious.

Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:




--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr
MT-118 @moduon

### CODE REVIEW

File: doc/cla/corporate/tecnativa.md [hunk 1]
```diff
@@ -16,7 +16,6 @@ Pedro M. Baeza pedro.baeza@tecnativa.com https://github.com/pedrobaeza
 Rafael Blasco rafael.blasco@tecnativa.com https://github.com/rafaelbn
 Sergio Teruel sergio.teruel@tecnativa.com https://github.com/sergio-teruel
 Carlos Dauden carlos.dauden@tecnativa.com https://github.com/carlosdauden
-Jairo Llopis jairo.llopis@tecnativa.com https://github.com/yajo
```
[the remark this answers is not in the document]
Comment (Yajo) [reply doc/cla/corporate/tecnativa.md:19]: Thanks, fixed.

**Comment by mart-e** [comment]
Oh actually, it was also reported via a support ticket and fixed at 7b3b623cf3359ac1eea537177f7f963e81403a01
If you wish, I can just merge the CLA commit if you remove the one in fetchmail.

**Comment by Yajo** [comment]
Oh! OK, thanks.

**Comment by mart-e** [comment]
@robodoo r+

**Comment by MiquelRForgeFlow** [comment]
It seems by error, @Yajo deleted the commit of the fix.

**Pull request closed without merging**